base/src/main/java/org/aya/core/term/Term.java

```diff
@@ -122,15 +122,8 @@ public sealed interface Term extends AyaDocile, Restr.TermLike<Term> permits Cal
       }
       case CallTerm.Prim prim -> {
         var args = prim.args().map(arg -> arg.descent(f));
-        yield switch (prim.id()) {
-          case IMIN -> PrimTerm.Mula.and(args.first().term(), args.last().term());
-          case IMAX -> PrimTerm.Mula.or(args.first().term(), args.last().term());
-          case INVOL -> PrimTerm.Mula.inv(args.first().term());
-          default -> {
-            if (args.sameElements(prim.args(), true)) yield prim;
-            yield new CallTerm.Prim(prim.ref(), prim.ulift(), args);
-          }
-        };
+        if (args.sameElements(prim.args(), true)) yield prim;
+        yield new CallTerm.Prim(prim.ref(), prim.ulift(), args);
```

Comment:
What if the args are not reduced but the head is an interval call?

Reply:
No.

Reply:
Most likely. But how to deal with it?

Reply:
I don't think it's here bc unfolding happens in `Expander` after this descent